src/store/modules/app-gateway-svc/index.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from 'vuex';
+import { state } from './states';
+import * as actions from './actions';
+import { mutations } from './mutations';
+import { ModuleState } from './types';
+import { RootState } from '../../root-types';
+
+// namespace name is set to the module name: appGateway
+const namespaced: boolean = true;
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Maybe?
```ts
const appGateway: Module<ModuleState, RootState> = {
  namespaced,
  state,
  actions,
  mutations,
};

export default appGateway;
```

Reply:
I read that we should avoid using export defaults: https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html

Reply:
the eslint says something different. i think people have different opinions. if we are going to use a linter, might as well stick to it?